docs/catalog/cloud/encrypt-what-is-necessary

```diff
@@ -0,0 +1,23 @@
+### Description
```

Comment:
Suggested header tags.

---
version: 1.0
submitted_by: @gopaltayal 
published_date: TBD
category: cloud
description: Data protection through encryption is a crucial aspect of our security measures. However, the encryption process can be resource-intensive at multiple levels.
tags:
- cloud
- security
---